packages/pages/src/index.ts

```diff
@@ -2,3 +2,5 @@ export * from "./dev/dev";
 export * from "./common/src/template/head";
 export * from "./common/src/template/paths";
 export * from "./common/src/template/types";
+export * from "./util";
```

Comment:
These shouldn't be necessary

Reply:
I was having trouble using the runTime stuff without this.  Do you want people to import it from a sub-package?

Reply:
specifically when I was trying to use it directly in a test site.  Do you want the usage to be: 
``` js
import { getRuntime} from "@yext/pages/util"
```
as opposed to:
``` js
import { getRuntime} from "@yext/pages"
```

Reply:
The package.json types and exports defines how to use the import. This index.ts is the entrypoint to the pagesjs dev server.